chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/item/README.md

# @vaadin/item

> A web component used to create items in a list-box, context-menu or select components.

Reviewer: ```suggestion
A web component for displaying items in list-box, context-menu or select components.
```

Author: Done.
